lib/versioning/VersioningRequestProcessor.ts

```diff
@@ -310,14 +310,14 @@ export default class VersioningRequestProcessor {
         nextValue?: string;
     }) {
         const { db, key } = request;
-        logger.info('start repair process', { request });
+        logger.info('start repair process');
         this.writeCache.get({ db, key }, logger, (err, value) => {
             // error or the new version is not a place holder for deletion
             if (err) {
-                return logger.info('error repairing', { request, error: err });
+                return logger.info('error repairing', { error: err });
```

Comment:
It seems so, I will turn to error, though I'll add a specific case for `ObjNotFound` and log in debug since it may happen if the PHD was deleted (by the user deleting the object) before it got repaired.